src/components/Article/Article.tsx

```diff
@@ -37,10 +38,7 @@ function renderCompetenceGoals(
     }) => ReactNode)
   | null {
   // Don't show competence goals for topics or articles without grepCodes
-  if (
-    !isTopicArticle &&
-    (article.competenceGoals?.length || article.coreElements?.length)
-  ) {
+  if (!isTopicArticle && article.competenceGoals?.length) {
```

Comment:
Et spørsmål forresten: Noen grunn til at denne if-statementen har blitt redusert?

Reply:
Mjaaaa, ish. Jeg mistenkte at vi kanskje ikke trengte den, samt at vi det blir litt teit å gjøre en hel haug med ekstra API-kall kun for å hente ut `coreElements`.length.